docs/manifest/sourcelocation.md

```diff
@@ -1,13 +1,15 @@
 ---
 title: SourceLocation element in the manifest file
 description: The SourceLocation element specifies the source file locations for your Office Add-in.
-ms.date: 06/14/2022
+ms.date: 12/01/2023
 ms.localizationpriority: medium
 ---
 
 # SourceLocation element
 
-Specifies the source file locations for your Office Add-in as a URL between 1 and 2018 characters long. The source location must be an HTTPS address, not a file path.
+Specifies the source file locations for your Office Add-in as a URL between 1 and 2018 characters long. The source location must be an HTTPS address, not a file path. 
+
+The source location can be any website, and you can use any directory structure you want. Typically, all the files in the add-in, except the manifest, are deployed to the source location. In the case of a single-page application, there is usually just one file. You can host some files on a CDN, or other domain, provided that you register the domain with an [AppDomain](appdomain.md) element. You would include a copy of the manifest at your source location only if your add-in needs to read a manifest value at runtime. 
```

Comment:
I'm a little confused by the comment "you can use any directory structure you want." Since the source location must be an HTTPS URL, what are we clarifying?

Reply:
The user who created the verbatim wanted to know if there was a required directory structure. This was my way of saying that no particular structure is required. I've reworded to emphasize that.

Reply:
Thanks! The reworded version makes more sense to me.